### DISCUSSION TITLE
Batch create files

### DISCUSSION THREAD

**Discussion by cem-al** [sxyazi/yazi#874] [1. Q&A]
Is there a way to batch create files, in a similar way to the batch rename functionality?

**Comment by AnirudhG07** [comment]
How are you imagining that? With keybinding, opening your editor, writing each file name or folder name(with /) and then closing will make the required files?

**Reply by cem-al** [reply]
Yes, exactly that ☺️

**Comment by xfzv** [comment]
Would be nice to have, indeed. I'd map it to `"A"`, considering `"a"` is the current default to create a single file or directory:

https://github.com/sxyazi/yazi/blob/9d0ef9a5dc84fb567df0154140f961d57a4bf6c6/yazi-config/preset/keymap.toml#L76

**Comment by SolitudeSF** [comment]
this doesnt require any file manager infrastructure, so it can easily be just an external program.

**Comment by sxyazi** [comment]
Closing in favor of https://github.com/sxyazi/yazi/issues/1974
